.github/workflows/audits.yml

```diff
@@ -92,7 +93,7 @@ jobs:
             *.cache-from=type=gha
             *.cache-to=type=gha,mode=max
       - name: Start
-        run: yarn workspace ${{ matrix.workspace }} start -d --wait
```

Comment:
Why?

Reply:
I was trying to figure out what was happening in the setup and was longing for clarity. I generally prefer the long form of CLI options unless working interactively.

Reply:
Ok I understand, but I prefer small changesets. :D